sway/config/output.c

```diff
@@ -239,6 +241,12 @@ struct output_config *store_output_config(struct output_config *oc) {
 		oc->transform, oc->background, oc->background_option, oc->power,
 		oc->max_render_time);
 
+	if (wildcard) {
+		merge_wildcard_on_all(oc);
+	} else {
+		oc = merge_id_on_name(oc);
+	}
```

Comment:
Not sure the ordering of this wrt. the logic above matters…? Maybe it does?

Reply:
Store_output_config is still weird. I feel like it would make more sense to:
1. Merge the config with anything existing (once!) and return the resulting config for callers to use
2. Otherwise, update the name according to the `id_on_name` scheme, add the config to the list and return it.

(I haven't spent any braincycles on the wildcard side of things.)

Reply:
I gave the cleanup I had in mind a go in https://github.com/swaywm/sway/pull/8050.